.github/actions/bolt-build-base/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Bolt Build Setup Steps'
+description: 'Executes necessary commands to setup environment for the bolt build'
+runs:
+  using: "composite"
+  steps:
+    - name: Setup conan profile
+      shell: bash
+      run: |
+        conan profile detect
+        sed -i 's/gnu14/gnu17/g' ~/.conan2/profiles/default
```

Comment:
for gcc 12,  the default flag is gun17.   This line can be removed.

Reply:
Thanks for pointing that out! Fixed.